src/main/java/controller/room/command/NewRoomOnCommand.java

```diff
@@ -0,0 +1,26 @@
+package controller.room.command;
+
+import dto.RoomDto;
+import dto.UserDto;
+import service.GameRoomService;
+
+import java.util.List;
+
+public class NewRoomOnCommand implements Command {
+    private static final int ARGUMENT_SIZE = 0;
+
+    public NewRoomOnCommand(final List<String> arguments) {
+        validateArgumentSize(arguments);
+    }
+
+    private void validateArgumentSize(final List<String> arguments) {
+        if (arguments.size() != ARGUMENT_SIZE) {
```

Comment:
```suggestion
        if (!arguments.isEmpty()) {
```
아마 Command 구현체의 구조를 전체적으로 통일시키려고 노력을 하신 것 같아요. 정말 좋은 시도라고 생각해요!
하지만 그 말은 결국 다른 Command 구현체에 상수가 추가되거나 구조가 바뀔 때 다른 구현체도 바꿔야해요. 🤔

Reply:
추가 기능을 만들면서 일관성에 다소 집착한 것 같아요. 피케이 말씀처럼 일관성을 지키려는 노력이 이점보다 더 큰 비용으로 이어질 수 있겠어요. 옛말에 중용이 중요하다고.. 우테코를 하면서 적당한 타협점을 찾는 방법을 배우는 것 같아요 😀

Reply:
맞아요. 유지보수하기 좋은 코드를 작성하는건 참 어려운 것 같아요.
저는 `개발을 하는 시점에서의 내 관점` 이 아닌 `1년 뒤, 모든 걸 잊어서 이 모든게 새롭게 느껴지는 내 관점`, `잘 모르는 상태에서 일부를 수정하려고 하는 팀원의 관점` 에서 보려고 노력하는 편이에요 (그렇다고 제가 엄청 잘한다는 것도 아니지만요 😂 )

저 관점에서 보면 특정 Command 를 수정할 경우, `개발을 하는 시점에서의 내 관점`을 온전히 다 이해하고 있을 가능성이 낮아요.
`얘는 왜 이렇게 짠거야?` 라고 생각하겠죠. 그 구조를 통일시키려고 다른 클래스를 함께 수정할 확률도 매우 낮구요.
그래서 그냥 처음부터 전체적인 일관성보다는 그 클래스의 상황에 더 잘어울리는 방식을 가져가는게 좋을 수도 있어요.

Reply:
> 1년 뒤, 모든 걸 잊어서 이 모든게 새롭게 느껴지는 내 관점, 잘 모르는 상태에서 일부를 수정하려고 하는 팀원의 관점

피케이 멋진 분이군요... 우테코 미션에 몰입할수록 지킥 참 어려운 것 같아요. 24시간 제 코드만 생각하고 있으니깐요. 앞으로 코드를 작성하면서 유념할게요!